docs/book/integrations.md

```diff
@@ -0,0 +1,43 @@
+---
+description: Use these tools out-of-the-box with ZenML.
+---
+
+# Integrations
+
+**ZenML** integrates with many different third-party tools.
+
+Once code is organized into a ZenML pipeline, you can supercharge your ML development with powerful integrations on multiple [MLOps stacks](core-concepts.md). There are lots of moving parts for all the MLOps tooling and infrastructure you require for ML in production and ZenML aims to bring it all together under one roof.
+
+We currently support [Airflow](https://airflow.apache.org/) and [Kubeflow](https://www.kubeflow.org/) as third-party orchestrators for your ML pipeline code. ZenML steps can be built from any of the other tools you usually use in your ML workflows, from [`scikit-learn`](https://scikit-learn.org/stable/) to [`PyTorch`](https://pytorch.org/) or [`TensorFlow`](https://www.tensorflow.org/).
+
+![ZenML is the glue](assets/zenml-is-the-glue.jpeg)
+
+These are the third-party integrations that ZenML currently supports:
+
+| Integration        | Status | Implementation Notes                       |
+| ------------------ | ------ | ------------------------------------------ |
+| Apache Airflow     | ✅      | Works for local environment                |
```

Comment:
One thing: Multiple integrations may have different dimensions.  E.g. Airflow can be used only as an orchestrator but it has a dimension of being `local` or `cloud`. Similarly plotly, dash, and graphviz are `visualizers` integrations but only for `Pipeline` and `PipelineRun` visualization objects. Is there an idea in the room how to showcase what integrations actually do and what a `tick` means? Perhaps we can add a description column with a detailed description and links to what it is?

Reply:
Like a kind of category? Yes I can do that.